### ISSUE TITLE
permissions failure on `python setup.py install`

### ISSUE THREAD

**Issue by davharris** [weecology/retriever#403]
If I don't use sudo, I get the following message as part of the output:

```
writing requirements to retriever.egg-info/requires.txt
error: [Errno 13] Permission denied: 'retriever.egg-info/requires.txt'
```

If we want users to use sudo, I think we should add it to the readme.

**Comment by ethanwhite** [comment]
Good point @davharris.

Getting this right is a little tricky because it depends on the users system configuration:

* conda/Anaconda: no sudo needed
* Windows: no sudo needed (and if you include sudo it won't work)
* OS X/Linux non-conda: sudo needed, or use `python setup.py install --user`

but I agree that being clearer would be useful.

Can you check and see what running `pip install .` does on your system (the dot is part of the command). I'm curious to see if it automatically does a `--user` install if you don't give it `sudo`.

**Comment by davharris** [comment]
I’m on Anaconda, though?

	davidharris$ python --version
	Python 2.7.11 :: Anaconda 2.4.0 (x86_64)

`Pip install .` works without sudo, as does `pip install git+URL`. 

`python setup.py install --user` does not work (same permission denied error).

**Comment by ethanwhite** [comment]
Interesting, so how Anaconda behaves is also OS dependent.

Do you think that adding a short statement that `sudo` may be necessary depending on your system would be sufficiently clear?

**Comment by davharris** [comment]
I think so, yeah. 

Dave

**Comment by ethanwhite** [comment]
OK, great. Thanks again for the feedback. I just opened a PR to fix this (#411).
